src/EditorFeatures/Core.Wpf/InlineHints/InlineHintsTagger.cs

```diff
@@ -154,7 +155,8 @@ public IReadOnlyList<TagSpan<IntraTextAdornmentTag>> GetTags(NormalizedSnapshotS
                     // active view and disregards and requests for tags not in that view
                     var fullSpan = new SnapshotSpan(snapshot, 0, snapshot.Length);
                     var tags = _tagAggregator.GetTags(new NormalizedSnapshotSpanCollection(fullSpan));
```

Comment:
i'm a bit confused by the logic below.  we are calling GetTags with the current snapshot we care about.  so all the tags we get back are for that snapshot right?  so why do we need to do things like mapping/tracking.  also, how could we get a tagspan that has multiple spans given it was already computed for this snapshot begin with.